### PR TITLE
Errata print01-ch06-buckets-visibility

### PR DESCRIPTION
**Pull request by joelposti**
Minor grammar-related fix for chapter 6 page 89. The original sentence seems to be missing a genitive 's. I propose the sentence to be expressed differently.

### CODE REVIEW

File: _errata/print01-ch06-buckets-visibility.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+chapter: 6
+page: 89
+kind: grammar
+reporter: Joel Posti
+date: 2023-09-13
+---
+
+The second sentence of the second paragraph says somewhat unclearly:
```
Comment (jonhoo) [inline _errata/print01-ch06-buckets-visibility.md]:
```suggestion
The second sentence of the second paragraph says:
```

File: _errata/print01-ch06-buckets-visibility.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+> We could fix this by making the `buckets` field visibility `pub(crate)`
+
+The sentence would be easier to understand if it were expressed like so:
```
Comment (jonhoo) [inline _errata/print01-ch06-buckets-visibility.md]: Which is accurate, but hard to follow. It should instead say:

**Review by jonhoo — APPROVED** [review]
Thanks! I think this _is_ technically acceptable grammar, but I agree it's a little hard to follow.

**Comment by joelposti** [comment]
I made your suggested changes.

**Review by jonhoo — APPROVED** [review]
